src/go/k8s/cmd/main.go

```diff
@@ -320,6 +317,20 @@ func main() {
 			os.Exit(1)
 		}
 
+		if unbindPVCsAfter <= 0 {
```

Comment:
I'm not crazy about 0 and negative numbers being the "disabled" case for the controller, but it does save having a separate bool flag. Plus, a user shouldn't ever see it AFAICT.

Reply:
I like to avoid flag duplication and additional validation when possible. 0 or a negative time would be an invalid for the "timeout" field it's used for. Instead of throwing an error, it's easier to disable the controller. And as you point out, this isn't a user facing flag.